### ISSUE TITLE
Leftover references to 14.1 in codebase

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7766]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7766|
|Type|release blocker: regression|
|Created|2015-02-02 22:44:16Z|
|Branch|https://github.com/twisted/twisted/tree/memory-hole-14.1-7766|

These should be changed to 15.0 for 15.1.

<details><summary>Searchable metadata</summary>

```
trac-id__7766 7766
type__release_blocker__regression release blocker: regression
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_memory_hole_14_1_7766 branches/memory-hole-14.1-7766
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1422917056342432 1422917056342432
changetime__1445217222500211 1445217222500211
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

I'll fix this in some down time.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [43830]) Branching to memory-hole-14.1-7766.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl removed owner|
|-|-|

Fixed, putting up for review, buildbots spun, etc. :)

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_4"></a>|@wsanchez set owner to @hawkowl|
|-|-|

Well that was an easy review.  Go forth and merge!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [43844]) Merging memory-hole-14.1-7766: Remove references to 14.1 in the codebase.

Author: hawkowl
Reviewer: wsanchez
Fixes: [#7766](https://github.com/twisted/twisted/issues/7766)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
